Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -80,7 +80,7 @@ class BrowserViewController: UIViewController {
     fileprivate var pasteAction: AccessibleAction!
     fileprivate var copyAddressAction: AccessibleAction!
 
-    fileprivate weak var tabTrayController: TabTrayController!
+    weak var tabTrayController: TabTrayController!
```

Comment:
Not a change you made but this being a forced unwrap weak variable is a red flag. We should change this to be an optional.